cmd/control/os.go

```diff
@@ -216,6 +216,16 @@ func startUpgradeContainer(image string, stage, force, reboot, kexec bool, upgra
 		}
 	}
 
+	fmt.Printf("Using %s\n", image)
```

Comment:
"Using" sort of reads as if that's the version that's currently running. I think "Upgrading to" was a bit more clear.